services/src/contexts/migrations/migration_0015_log_quota.rs

```diff
@@ -12,15 +9,32 @@ pub struct Migration0015LogQuota;
 #[async_trait]
 impl Migration for Migration0015LogQuota {
     fn prev_version(&self) -> Option<DatabaseVersion> {
-        Some(Migration0014MultibandColorizer.version())
+        None // must have migrated to 0015 with old version
     }
 
     fn version(&self) -> DatabaseVersion {
         "0015_log_quota".into()
     }
 
-    async fn migrate(&self, _tx: &Transaction<'_>) -> Result<()> {
-        // Nothing to do here, only in Pro
+    async fn migrate(&self, tx: &Transaction<'_>) -> Result<()> {
+        let config = crate::config::get_config_element::<crate::config::Postgres>()?;
+
+        let schema_name = &config.schema;
+
+        if schema_name != "pg_temp" {
+            tx.batch_execute(&format!("CREATE SCHEMA IF NOT EXISTS {schema_name};",))
+                .await?;
+        }
+
+        tx.batch_execute(include_str!("migration_0015_snapshot.sql"))
+            .await?;
+
+        tx
+            .execute(
+                "INSERT INTO geoengine (clear_database_on_start, database_version) VALUES ($1, '0000_initial');",
```

Comment:
why version 0000_inital and not 0015?

Reply:
[b697ee8](https://github.com/geo-engine/geoengine/pull/1009/commits/b697ee8a8194c602e1fb8bde36af9ba6b61b9fe1)